Feature/japx examples (#25)
infinum/iOS-Nuts-And-Bolts: Catalog/Examples/Networking/Japx/README.MD

# Networking with Japx

[Japx](https://github.com/infinum/Japx) is lightweight JSON:API parser that flattens complex JSON:API structure and turns it into simple JSON and vice versa.

## Example

> This example will cover how to handle basic CRUD (Create, Read, Update, Delete) operations with Japx because API expects JSON:API format. 

Reviewer: ```suggestion
This example will cover how to handle basic CRUD (Create, Read, Update, Delete) operations with Japx and JSON:API. 
```
